packages/core/src/components/hotkeys/hotkey.tsx

```diff
@@ -61,6 +62,12 @@ export interface IHotkeyProps {
      */
     stopPropagation?: boolean;
 
+    /**
+     * Space-delimited string of class names applied to the
+     * `KeyCombo` contained in the hotkey.
+     */
+    keyComboClassName?: string;
```

Comment:
why do we need this?

Reply:
we don't strictly need it and it's probably safe to remove. shall i do that?